### ISSUE TITLE
Console log has wrong font color

### ISSUE THREAD

**Issue by minemeraj** [jenkinsci/material-theme-plugin#16]
### Jenkins and plugins versions report

<details>
  <summary>Environment</summary>

  
  ```text
Jenkins: 2.348
OS: Linux - 3.10.0-1160.42.2.el7.x86_64

simple-theme-plugin:103.va_161d09c38c7
material-theme:0.4.1

  ```

</details>

### What Operating System are you using (both controller, and any agents involved in the problem)?

Linux - 3.10.0-1160.42.2.el7.x86_64

### Reproduction steps

Open the console log of any build

### Expected Results

<img width="462" alt="image" src="https://user-images.githubusercontent.com/2042835/169281235-91b7dd64-893a-404e-95f2-23f046e977b8.png">

### Actual Results

<img width="1544" alt="image" src="https://user-images.githubusercontent.com/2042835/169281320-94f02741-b1ad-4292-8a0e-f4d3486f5fa5.png">

### Anything else?

it might be related with:
https://github.com/jenkinsci/jenkins/pull/6339
https://github.com/jenkinsci/jenkins/pull/6338

**Comment by jonesbusy** [comment]
Hi,

Happens now with latest LTS 2.346.1 and latest plugins.

Workaround for me is to fix the CSS with 'simple-theme-plugin'

Example of JCasC

```
simple-theme-plugin:
  elements:
    - cssText:
        text: |-
          pre.console-output {
            color: #FFFFFF !important;
          }
          .cbwf-stage-logs .log-details .console-output {
            color: #000000 !important;
          }
```

Best regards,

**Comment by offa** [comment]
Fixed in 0.5.0

**Comment by acsipak** [comment]
Shouldn't there be a reference to this issue (#16) in the [0.5.0 release notes](https://github.com/jenkinsci/material-theme-plugin/releases/tag/v0.5%2C0)?

**Comment by offa** [comment]
The linked PR is listed instead, which has a similar title.

**Comment by timbrown5** [comment]
Should be fixed in 0.5.0-rc83.6557e02980de (please reoppen if not).

**Comment by minemeraj** [comment]
The issue is solved! Thanks!
